grammars.yml

```diff
@@ -793,6 +793,13 @@ vendor/grammars/language-ti-basic:
 - source.8xp
 vendor/grammars/language-toc-wow:
 - source.toc
+vendor/grammars/language-tools:
```

Comment:
Since [vuejs/language-tools](https://github.com/vuejs/language-tools) repo contains multiple packages, should this be `vendor/grammars/language-tools/extensions/vscode/syntaxes`?

I just left what the `add-grammar --replace` command gave me intact but wasn't really sure...

Reply:
No. What the script has done is correct. We clone the entire repo as a submodule to that directory.